st2reactor/st2reactor/timer/base.py

```diff
@@ -132,8 +134,9 @@ def _emit_trigger_instance(self, trigger):
             'executed_at': str(utc_now),
             'schedule': trigger['parameters'].get('time')
         }
+
         trace_context = TraceContext(trace_tag='%s-%s' % (self._get_trigger_type_name(trigger),
-                                                          trigger.name))
+                                                          trigger.get('name', uuid.uuid4().hex)))
```

Comment:
No. not possible. 

This trigger thing is such a disaster that I would hate for it to blow up for dumb things.